src/main/java/seedu/address/MainApp.java

```diff
@@ -57,13 +62,14 @@ public void init() throws Exception {
         UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
         AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
-        storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        EventBookStorage eventBookStorage = new JsonEventBookStorage(userPrefs.getEventBookFilePath());
+        storage = new StorageManager(addressBookStorage, eventBookStorage, userPrefsStorage);
 
         initLogging(config);
 
         model = initModelManager(storage, userPrefs);
-
-        logic = new LogicManager(model, storage);
+        eventModel = initModelManager(storage, userPrefs);
```

Comment:
whats the diff between eventModel and model? uses the same inputs.

Reply:
should be using initEventModelManager right?

Reply:
Cos storage contains both ab and eb, so can reuse initmodelmanager

Reply:
Logic manager needs both to process them differently